src/pyskindose/table_data/generated_hvl_spekpy_data.py

```diff
@@ -0,0 +1,187 @@
+import numpy as np
+import spekpy as sp  # Note: Not included in PySkinDose,
+import pandas as pd
+import os
+from pathlib import Path
+
+
+def generate_spekpy_data(
```

Comment:
This function should have type hints and a Docstring added

Reply:
done! 

file name is  changed to generate_hvl_data